api/src/main/java/com/cloud/vm/snapshot/VMSnapshot.java

```diff
@@ -31,7 +31,7 @@
     enum State {
         Allocated("The VM snapshot is allocated but has not been created yet."), Creating("The VM snapshot is being created."), Ready(
                 "The VM snapshot is ready to be used."), Reverting("The VM snapshot is being used to revert"), Expunging("The volume is being expunging"), Removed(
-                "The volume is destroyed, and can't be recovered."), Error("The volume is in error state, and can't be recovered");
+                "The volume is destroyed, and can't be recovered."), CreatingKVM("Creating KVM VM snapshot"), Error("The volume is in error state, and can't be recovered");
```

Comment:
VM snapshot states should be also  generic. Can this be addressed in any other way?

Reply:
I'll check if I can omit those changes, thanks @sureshanaparti

Reply:
removed the new state and events related to it